added nitesh to contributors

sudonitesh opened the pull request.

palash25 commented:
@itsmepiyush2 check for grammatical errors

and try to review all the other upcoming contributor PRs

itsmepiyush2 commented on _contributors-app/sudonitesh.md:
```diff
@@ -0,0 +1,36 @@
+   - [Wiki Viewer](https://github.com/sudonitesh/fcc-wikipedia): Wikipedia Viewer.
+
+volunteer_experience: |
+ - Third in team based GenHax hackathon. Made a web-app to digitalise garbage-collection of the country.
```
A better statement: "I participated in a team based hackathon "GenHax" where we secured the third position. We made a web-app to help digitalise India's garbage collection."

knrt10 commented:
@sudonitesh please remember to make one commit only. Didn't you read [meta](https://github.com/p-society/meta). Please squash your commits

sudonitesh commented:
@knrt10  I've squashed the commits locally, Now what to do? 

nitesh@Nitesh-PC MINGW64 ~/Desktop/p-society.github.io (profile)
$ git rebase -i profile
Successfully rebased and updated refs/heads/profile.
nitesh@Nitesh-PC MINGW64 ~/Desktop/p-society.github.io (profile)
$ git log --oneline
bb0b431 (HEAD -> profile) add nitesh to contributors after update
6037255 (origin/master, origin/HEAD) added social media icons to contact page
27c2edd added regex test
....
nitesh@Nitesh-PC MINGW64 ~/Desktop/p-society.github.io (profile)
$ git log --author=sudonitesh
commit bb0b431756ce2f093b22444dacf994fd03436a08 (HEAD -> profile)
Author: sudonitesh <nktosni@gmail.com>
Date:   Mon Aug 27 22:58:39 2018 +0530

    add nitesh to contributors after update

    added nitesh to contributors

    updated volunteer_experience

sudonitesh commented:
I think its done. Please check

sibasish14 commented on _contributors-app/sudonitesh.md:
```diff
@@ -0,0 +1,36 @@
+
+github_url: https://github.com/sudonitesh
+linkedIn_url: https://www.linkedin.com/in/sudonitesh/
+twitter_url: https://twitter.com
```
@sudonitesh give your correct Twitter url.

sudonitesh replied:
Have you seen me on twitter?!! I don't have one..

sibasish14 commented:
Then remove the link. Why do you want to redirect someone to Twitter when you don't have one?😉

palash25 commented:
@sudonitesh good work you know how to squash now.

But why do you repeat the same shortlog twice in your commit message. Please read how to make good commit message the link is in the Meta README

sudonitesh commented:
@palash25 ya sure i'll . do i need to make any change in this pr?

palash25 commented:
Just told you remove the second occurence of the shortlog and push, then I will merge

Also shibasis gave you a review and you haven't addressed it.

palash25 commented:
Which second commit? You only have one

palash25 commented:
Change this message 

```
added nitesh to contributors
added nitesh to contributors

updated volunteer_experience
```

to this

```
added nitesh to contributors

updated volunteer_experience
```

And also remove the twitter link if you don't have a twitter

sudonitesh commented:
@palash25 please review
